pinot-controller/src/main/resources/app/requests/index.ts

```diff
@@ -89,8 +93,8 @@ export const putSchema = (name: string, params: string, reload?: boolean): Promi
 export const getSegmentMetadata = (tableName: string, segmentName: string): Promise<AxiosResponse<SegmentMetadata>> =>
   baseApi.get(`/segments/${tableName}/${segmentName}/metadata?columns=*`);
 
-export const getTableSize = (name: string): Promise<AxiosResponse<TableSize>> =>
-  baseApi.get(`/tables/${name}/size`);
+export const getTableSize = (name: string, detailed: boolean): Promise<AxiosResponse<TableSize>> =>
+  baseApi.get(`/tables/${name}/size?detailed=${detailed}`);
```

Comment:
detailed can be optional param, default value would be false
this way you don't have to pass false to every function call, only function that requires detailed data should pass true.

```suggestion
export const getTableSize = (name: string, detailed: boolean = false): Promise<AxiosResponse<TableSize>> =>
  baseApi.get(`/tables/${name}/size?detailed=${detailed}`);
```